docs/src/dialogs.md

```diff
@@ -126,3 +126,52 @@ Page.Dialog += async (_, dialog) =>
 };
 await Page.CloseAsync(new() { RunBeforeUnload = true });
 ```
+
+## Print dialogs
+
+In order to assert that a print dialog via [`window.print`](https://developer.mozilla.org/en-US/docs/Web/API/Window/print) was opened, you can use the following snippet:
+
+```js
+await page.evaluate('(() => {window.waitForPrintDialog = new Promise(f => window.print = f);})()');
```

Comment:
- evaluate: needs to be after navigation
- addInitScript: needs to be before navigation

Which one is better? evaluate and be explicit?

Reply:
Discussed offline: evaluate is simpler.